sdcm/nemesis.py

```diff
@@ -1005,9 +1005,11 @@ def _parse_toppartitions_output(output):
 
         def generate_random_parameters_values():  # pylint: disable=invalid-name
             ks_cf_list = get_non_system_ks_cf_list(self.loaders.nodes[0], self.cluster.nodes[0])
-            if not ks_cf_list:
-                raise Exception("No user defined keyspace.columnfamily")
-            ks, cf = random.choice(ks_cf_list).split('.')
+            try:
+                ks, cf = random.choice(ks_cf_list).split('.')
+            except IndexError:
+                self.log.error('User-defnied Keyspace and ColumnFamily are not found %s.', ks_cf_list)
+                raise Exception('User-defnied Keyspace and ColumnFamily are not found')
```

Comment:
need exception details and more debugging to log in order to understand why we fail

Reply:
Adding details if error happened to log and to raising Exception